RecoLocalTracker/SiPixelRecHits/plugins/PixelCPEFastESProducer.cc

```diff
@@ -92,6 +92,8 @@ void PixelCPEFastESProducer::fillDescriptions(edm::ConfigurationDescriptions& de
   // specific to PixelCPEFastESProducer
   desc.add<std::string>("ComponentName", "PixelCPEFast");
   desc.add<edm::ESInputTag>("MagneticFieldRecord", edm::ESInputTag());
+  desc.addOptional<bool>("useLAAlignmentOffsets", false)->setComment("deprecated");
```

Comment:
@ianna why this change?

Reply:
I think rebase didn't take into an account reverted changes :-(